pom.xml

```diff
@@ -92,12 +92,6 @@
       <artifactId>mailer</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.main</groupId>
-      <artifactId>jenkins-test-harness-tools</artifactId>
```

Comment:
Could simply keep this, and exclude its dep on JTH.

Reply:
lets do that instead